javascript/elements/updates_for_element.js

```diff
@@ -134,6 +134,10 @@ export default class UpdatesForElement extends SubscribingElement {
     )
   }
 
+  get query () {
+    return `updates-for[identifier="${this.getAttribute('identifier')}"]`
```

Comment:
Can't you use `this.identifier` here too?
```suggestion
    return `updates-for[identifier="${this.identifier}"]`
```

Reply:
🙏